### DISCUSSION TITLE
除了base以外 剩下的服务开启seata 无法开启

### DISCUSSION THREAD

**Discussion by teng-jy** [zuihou/zuihou-admin-cloud#164] [02.提问]
你好， 问个问题我这个有base，system两个服务 其中配置seata是一样的 但是我的base 可以进SeataAutoConfiguration类 并且在ConfigurationFactory中的buildConfiguration方法中拿到type为nacos，但是我的system 启动就不进入SeataAutoConfiguration，并且在 buildConfiguration方法的type 为null

![%U@HUG6EQNST9FTSF9H`9)F](https://user-images.githubusercontent.com/62694039/180716191-540fe209-c6dc-4d85-9e80-1d7e6c82437b.png)
![UD7UM2CQ}6MR3G M @FDAPE](https://user-images.githubusercontent.com/62694039/180716206-aa591bbe-aa0e-4836-97ec-d034b5c5f3c5.png)

![%PLY~%_V I6D}ICRQPB`V20](https://user-images.githubusercontent.com/62694039/180716225-04f448dd-21f9-444d-bb2e-ab98813253fc.png)

**Comment by zuihou** [comment]
https://github.com/zuihou/lamp-cloud/discussions/163
